apps/backoffice-v2/src/common/hooks/useHomeLogic/useHomeLogic.ts

```
@@ -1,39 +1,174 @@
+  };
+
+  const getRiskColor = (risk: string) => {
+    return RISK_LEVEL_COLORS[risk.toLowerCase() as keyof typeof RISK_LEVEL_COLORS] || '#65afff';
+  };
+
+  // Pre-calculate configs with empty data to ensure hooks are called unconditionally
+  const statusConfig = useMemo(() => {
+    if (!currentStats) return {};
+    return currentStats.casesByStatus.reduce(
+      (acc, curr) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)